protocols/grpc/hyperstart.proto

```diff
@@ -46,6 +47,14 @@ message StartContainerRequest {
 	string container_id = 1;
 }
 
+message SignalContainerRequest {
+	string container_id = 1;
+	uint32 signal = 2;
+	// When all is set to true, the signal will be
+	// sent to all processes inside the container.
+	bool all = 3;
```

Comment:
We don't need this flag(bool all) at all. If the only the init process is signaled, we can just use the SignalProcess().

SignalProcess() only kill the root processes of the init/exec process. So I admit that SignalContainer() might be needed if SignalContainer() kills not only the root processes but all the natural spawned linux process,  only when libcontainer support this operation.